### PR TITLE
WD-12749 - rebrand /core/reatures/recovery

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand /core/features/recovery page. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14238.demos.haus/core/features/recovery)
- [figma](https://www.figma.com/design/5kDdBtPRqthyPz263YV91U/24.04-Ubuntu-Core-rebrand?node-id=2005-3267&t=6F9qsf1L0Y031BcA-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12749](https://warthogs.atlassian.net/browse/WD-12749)

[WD-12749]: https://warthogs.atlassian.net/browse/WD-12749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
@lizzochek 

Hero image should be smaller, I uploaded [here](https://assets.ubuntu.com/v1/a54f664a-device%20recovery%20hero%20img
Screenshot 2024-08-28 at 17 02 16
.png)

Could you do a `<br>` here after recovery 
Screenshot 2024-08-28 at 17 03 43

**Comment by lizzochek** [comment]
@mattea-turic updated everything except hero image. are you sure it has to be smaller? it is 600px as in the figma and the new asset you uploaded has a market bg as needed

**Comment by mattea-turic** [comment]
@lizzochek I don't I described well, sorry. I meant the illustration rather than the image overall. 600px is good, but the illustration itself should have some whitespace around it so that it's not stretching to any of the edges in the container

**Comment by immortalcodes** [comment]
@mattea-turic The image is blur for me and might be  more blur on a higher resolution screen, could we replace it with a image of better resolution.
image

File: templates/core/features/recovery.html [hunk 1]
```diff
@@ -1,141 +1,174 @@
...
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <h2>Device recovery on Ubuntu Core</h2>
     </div>
-    <div class="p-card col-6 u-vertically-center">
-      <h3 class="p-card__title">Free for pre-certified boards</h3>
-      <p class="p-card__content">Recovery mode is available out of the box on <a href="/certified/iot">certified devices</a>, like the Raspberry Pi, at no additional cost. An enablement fee is required to fully certify Ubuntu Core on non-certified boards. </p>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="p-rule is-muted" />
```
Comment (immortalcodes) [inline templates/core/features/recovery.html:63]: p-rule class has been depreciated

File: templates/core/features/recovery.html [hunk 2]
```diff
@@ -1,141 +1,174 @@
...
-    <h2>How it works</h2>
-    <p class="p-heading--4">Ubuntu Core 20 introduces a secure recovery system.</p>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h3>Recovery mode</h3>
-      <p>Ubuntu Core offers a recovery mode that can be activated manually when booting, or remotely via API call. In recovery mode, Ubuntu Core presents a UI for users to enter recovery options. The operating system then proceeds to fetch the configuration files and snaps associated with a given configuration profile.</p>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="p-rule is-muted" />
```
Comment (immortalcodes) [inline templates/core/features/recovery.html:78]:
```suggestion
        <hr class="is-muted" />
```

File: templates/core/features/recovery.html [hunk 3]
```diff
@@ -1,141 +1,174 @@
...
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h3>Snapshots</h3>
-      <p>Multiple snapshots for the same device can be backed up in the recovery system. These snapshots reflect both configuration settings and the collection of snaps installed on the system. Device operators can create, name and save such snapshots in the recovery system. This enables a swift recovery of a desired system state when needed. Recovery can happen manually through a dedicated UI or remotely via an API call.</p>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="p-rule is-muted" />
```
Comment (immortalcodes) [inline templates/core/features/recovery.html:105]:
```suggestion
        <hr class="is-muted" />
```

File: templates/core/features/recovery.html [hunk 4]
```diff
@@ -1,141 +1,174 @@
...
+          <div class="p-image-container is-highlighted">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/683e6225-Image.png"
+                 alt="" />
+          </div>
+        </div>
+        <p>Get in touch with an Ubuntu security expert to discuss the advanced security requirements of your application.</p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/core/contact-us?product=core-recovery"
```
Comment (immortalcodes) [inline templates/core/features/recovery.html:158]:
```suggestion
          <a href="/core/contact-us"
```
js-invoke-modal class will invoke the required form here but if that doesn't happen then it should direct to contact page.

**Review by immortalcodes — COMMENTED** [review]
Thanks for the PR. 
Minor comments left on code and a question  for Mattea, I would also suggest to use image tag instead of direct images link to benefit from CDN.

**Comment by lizzochek** [comment]
@immortalcodes about image tags - no, I am using new image container, so It does not need an image tag

**Review by immortalcodes — APPROVED** [review]

**Comment by mattea-turic** [comment]
@lizzochek I've added an updated image [here](https://assets.ubuntu.com/v1/08abf309-low-touch-device-maintenance.png). Thanks for pointing out the low-res issue @immortalcodes

**Pull request merged**